pcdsdevices/pneumatic.py

```diff
@@ -0,0 +1,97 @@
+"""
+Pneumatic Classes.
+
+This Module contains all the classes relating to Pneumatic Actuators
+"""
+
+from lightpath import LightpathState
+from ophyd import Component as Cpt
+from ophyd import EpicsSignal, EpicsSignalRO
+from ophyd.status import Status
+
+from pcdsdevices.interface import BaseInterface, LightpathMixin
+
+
+class BeckhoffPneumatic(BaseInterface, LightpathMixin):
+    """
+    Class containing basic Beckhoff Pneumatic support
+    """
+    lightpath_cpts = ['limit_switch_in', 'limit_switch_out']
+
+    # readouts
+    limit_switch_in = Cpt(EpicsSignalRO, ':PLC:bInLimitSwitch')
+    limit_switch_out = Cpt(EpicsSignalRO, ':PLC:bOutLimitSwitch')
+
+    retract_status = Cpt(EpicsSignalRO, ':bRetractDigitalOutput')
+    insert_status = Cpt(EpicsSignalRO, ':bInsertDigitalOutput')
+
+    # logic and supervisory
+    interlock_ok = Cpt(EpicsSignalRO, 'bInterlockOK')
+    insert_ok = Cpt(EpicsSignalRO, 'bInsertEnable')
+    retract_ok = Cpt(EpicsSignalRO, 'bretractEnable')
+
+    # commands
+    insert_signal = Cpt(EpicsSignal, 'CMD:IN')
+    retract_signal = Cpt(EpicsSignal, 'CMD:OUT')
+
+    # returns
+    busy = Cpt(EpicsSignalRO, ':bBusy')
+    done = Cpt(EpicsSignalRO, ':bDone')
+    reset = Cpt(EpicsSignal, ':bReset')
+    error = Cpt(EpicsSignalRO, ':PLC:bError')
+    error_id = Cpt(EpicsSignalRO, ':PLC:nErrorId')
```

Comment:
Are these not pytmc-generated? If they are, all these read-only signals should have a `_RBV` at the end of them or you'll get the setpoint at best (or a pv-not-found at worst)

Reply:
One way to handle this is to use the PytmcSignal https://github.com/pcdshub/pcdsdevices/blob/525a89eec8a54fd71d784052542daec8813a9d29/pcdsdevices/signal.py#L38

Reply:
I went with PytmcSingal. Thanks for catching that Ken and Zach.